emu/processes/wps_geospatial.py

```diff
@@ -0,0 +1,144 @@
+"""
+Process using an application/geo+json and 'image/tiff; application=geotiff' input. Used to test UI interactions.
+
+Author: Trevor James Smith
+"""
+from pywps import Process, ComplexInput, LiteralOutput, BoundingBoxOutput, ComplexOutput
+from pywps import FORMATS
+import logging
+
+LOGGER = logging.getLogger("PYWPS")
+
+__all__ = ["GeoData"]
+
+
+class GeoData(Process):
```

Comment:
I can strip down the examples a bit but I wanted to showcase a fairly simple read/write and export operation for users building off Emu. I can strip down the logic a bit but the Fiona/Rasterio API has been standard for years. Perhaps I can make it so that this example isn't tested if the dependencies aren't there?

Reply:
I've simplified it quite a bit, but it would be good for testing purposes to have something that showcases inputs and outputs of GeoJSON and GeoTIFF files. I elected to have the example data I made stay under `tests` as we don't necessary want it in the package (again, it's only for testing/development). I hope this is more in line with our goals.